data/json/itemgroups/books.json

```
@@ -239,17 +239,17 @@
       { "item": "holybook_bahai3", "prob": 5 },
       { "item": "holybook_bahai4", "prob": 5 },
       { "item": "holybook_bahai5", "prob": 5 },
-      { "item": "holybook_quran", "prob": 20 },
+      { "item": "holybook_quran", "prob": 500 },
       { "item": "holybook_hadith", "prob": 18 },
       { "item": "holybook_havamal", "prob": 8 },
-      { "item": "holybook_tanakh", "prob": 20 },
-      { "item": "holybook_talmud", "prob": 20 },
+      { "item": "holybook_tanakh", "prob": 500 },
+      { "item": "holybook_talmud", "prob": 500 },
```

```suggestion
      { "item": "holybook_talmud", "prob": 25},
```
Actual copies of the Talmud are very rare (it's specialized and _extremely_ expensive), usually only found in locations dedicated to Talmudic study, so regardless of the proportion of Jews in the Northeast this would still be very difficult to find.

I would bump `book_judaica_siddur` up to 500 in its place. That's the prayerbook for the daily service--every synagogue has hundreds of these and there's one or more in the home of every even moderately observant Jew (I have two different ones on my shelf right now).